Web platform release notes for Edge Beta 146 (#3719)
MicrosoftDocs/edge-developer: microsoft-edge/web-platform/release-notes/146.md

<!-- ------------------------------ -->
#### WebGPU Compatibility mode

WebGPU Compatibility mode adds an opt-in, lightly restricted subset of the WebGPU API, that's capable of running older graphics APIs such as OpenGL and Direct3D11.

By opting into this mode and following its constraints, you can extend the reach of your WebGPU application to many older devices that don't have the modern, explicit graphics APIs that core WebGPU requires.

> * For a simple app, the only required change is to specify the `"compatibility"` `featureLevel`<!-- todo: is featureLevel correct?  clarify --> when calling `requestAdapter()`. 

Author: ```suggestion
* For a simple app, the only required change is to set the `featureLevel` option to `"compatibility"` when calling `requestAdapter()`. 
```

<!-- ------------------------------ -->
#### WebGPU: Texture and Sampler `let` binding

> <!-- todo: resolve one-to-many ambiguity.  how many things go in 1 'let' declaration?  favor singular not plural -->

Reviewer: Resolve one-to-many ambiguity. How many things go in 1 'let' declaration? Favor singular not plural.

@captainbrosset

> WGSL now supports storing texture and sampler objects<!-- todo: singular --> in `let` declarations<!-- todo: singular --> via the `texture_and_sampler_let` language feature.

Author: ```suggestion
In WGSL, a `let` declaration can now store an object of type texture or sampler, thanks to the `texture_and_sampler_let` language feature.
```
